apollo-router/src/services/layers/persisted_queries/manifest_poller.rs

```diff
@@ -783,15 +782,14 @@ mod tests {
         let manifest_manager = PersistedQueryManifestPoller::new(
             Configuration::fake_builder()
                 .apq(Apq::fake_new(Some(false)))
-                .persisted_query(PersistedQueries::new(
-                    Some(true),
-                    Some(false),
-                    Some(PersistedQueriesSafelist::default()),
-                    Some(false),
-                    Some(vec![
-                        "tests/fixtures/persisted-queries-manifest.json".to_string()
-                    ]),
-                ))
+                .persisted_query(
+                    PersistedQueries::builder()
```

Comment:
Switching to using the builder seemed more legible if I had to update this anyway.